pkg/k8shandler/desirednodestate.go

```diff
@@ -342,14 +342,6 @@ func (cfg *desiredNodeState) getEnvVars() []v1.EnvVar {
 			Name:  "HEAP_DUMP_LOCATION",
 			Value: heapDumpLocation,
 		},
-		v1.EnvVar{
-			Name:  "NODE_QUORUM",
-			Value: strconv.Itoa(int(cfg.MasterNum/2 + 1)),
-		},
-		v1.EnvVar{
-			Name:  "RECOVER_EXPECTED_NODES",
-			Value: strconv.Itoa(int(cfg.DataNum)),
-		},
```

Comment:
so users will need to edit the configmap instead of being able to set these as env. vars. in the deployment?

Reply:
Maybe that's better since you can set it in one place with a configmap vs. each deployment (until we move to statefulset . . .)

Reply:
We don't allow modification of the configuration stored in the config map.
We run sha256 on the config map content and revert all changes made by 'someone else other than the operator'

What we do allow users to change though is `REPLICA_SHARDS`. And that's only because we aren't able to deliver `resiliency flavours` (no replication, some replication, full replication) in time for the release. @ewolinetz can say more about this.

Reply:
I don't see why we couldn't incorporate the `REPLICA_SHARDS` change as part of this.

Reply:
openshift/origin-aggregated-logging/pull/1468

Reply:
I put REPLICA_SHARDS in the same config map as PRIMARY_SHARDS